python/rmm/tests/test_rmm.py

```diff
@@ -23,6 +23,9 @@
 
 _driver_version = rmm._cuda.gpu.driverGetVersion()
 _runtime_version = rmm._cuda.gpu.runtimeGetVersion()
+_CUDAMALLOC_ASYNC_SUPPORTED = (_driver_version >= 11020) and (
```

Comment:
It has to `or` no(instead of `and`)?

Reply:
We need both conditions to hold, correct?

Reply:
Ah, never mind didn't notice `>=`

Reply:
Note this variable indicates whether `cudaMallocAsync` _is_ supported.

Reply:
> Note this variable indicates whether `cudaMallocAsync` _is_ supported.

Yup, missed this.